code/modules/mob/living/basic/bots/_bots.dm

```diff
@@ -735,11 +735,16 @@ GLOBAL_LIST_INIT(command_strings, list(
 	. = ..()
 	if(!. || isnull(client))
 		return FALSE
+	REMOVE_TRAIT(src, TRAIT_NO_GLIDE, INNATE_TRAIT)
+	speed = 2
```

Comment:
I've already changed medibots base speed to 2 (it was that originally), and this may effect other bots turned into simple bots

Reply:
yes this is meant to target all bots. the reason i changed their original speeds is because they were a bit too fast with the new AI controller